data/games/guilty_gear_xrd_rev_2/index.js

```diff
@@ -1,3 +1,29 @@
+import answer_images from './images/answer';
+import axl_images from './images/axl';
+import baiken_images from './images/baiken';
+import bedman_images from './images/bedman';
+import chipp_images from './images/chipp';
+import dizzy_images from './images/dizzy';
+import elphelt_images from './images/elphelt';
+import faust_images from './images/faust';
+import ino_images from './images/i-no';
+import jacko_images from './images/jack-o';
+import jam_images from './images/jam';
+import johnny_images from './images/johnny';
+import kum_images from './images/kum';
+import ky_images from './images/ky';
+import leo_images from './images/leo';
+import may_images from './images/may';
+import millia_images from './images/millia';
+import potemkin_images from './images/potemkin';
+import ramlethal_images from './images/ramlethal';
+import raven_images from './images/raven';
+import sin_images from './images/sin';
+import slayer_images from './images/slayer';
+import sol_images from './images/sol';
+import venom_images from './images/venom';
+import zato_images from './images/zato';
+
```

Comment:
I'm not wild about this but I didn't have any luck with like:
`data/games/guilty_gear_xrd_rev_2/images/index.js` made up of statements like:
```js
export answer_images from './images/answer';
...
```
and
`data/games/guilty_gear_xrd_rev_2/index.js` including:
```js
import {answer_moves} from './images';
...
```

If you can figure out how to make that work, it would be a lot cleaner. I'm just not familiar enough with js/node/react yet.